GNUmakefile

```diff
@@ -274,10 +270,7 @@ test-nomad: dev ## Run Nomad test suites
 		$(if $(ENABLE_RACE),-race) $(if $(VERBOSE),-v) \
 		-cover \
 		-timeout=15m \
-		$(GOTEST_PKGS) $(if $(VERBOSE), >test.log ; echo $$? > exit-code)
```

Comment:
I had some issues with this on circle and didn't fix it yet, will handle before merging this